app/actions/views/edit_profile.js

```diff
@@ -25,7 +25,21 @@ export function setProfileImageUri(imageUri = '') {
     };
 }
 
+export function removeProfileImage(user, success, error) {
+    return async (dispatch, getState) => {
+        const result = await setDefaultProfileImage(user)(dispatch, getState);
```

Comment:
Minor nit, but this can be changed to `const result = await dispatch(setDefaultProfileImage(user));` which also removes the need for the `getState` prop. They're practically equivalent, but `await dispatch` is a bit shorter and works a bit better than passing dispatch and getState in as arguments. We're intending to make this change everywhere, but we've just been doing it piece-by-piece for the time being.